packages/cloud_firestore/lib/src/document_reference.dart

```diff
@@ -42,6 +42,17 @@ class DocumentReference {
     );
   }
 
+  Future<DocumentSnapshot> get() {
+    return Firestore.channel.invokeMethod('DocumentReference#get',
+        <String, dynamic>{'path': path}).then((Map<String, dynamic> data) {
+      return new DocumentSnapshot._(
+          data['path'], data['data'], Firestore.instance);
```

Comment:
I would put a trailing comma here so it formats better

Reply:
Done! I'm still getting used to these trailing commas.